docs/WEBSOCKET_IMPLEMENTATION_REVIEW.md

```
@@ -0,0 +1,411 @@
+const unsubscribeContextChanged = chatService.onConversationContextChanged(
+  ({ conversationId, context }) => {
+    setConversationContext(conversationId, context);
+  },
+);
+
+const unsubscribeArchived = chatService.onConversationArchived(({ conversationId }) => {
+  archiveConversation(conversationId);
+});
+```
+
+### **File: `src/services/chatService.ts`**
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)